internal/cli/main.go

```diff
@@ -190,8 +191,23 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 			log.fatalErr(migraterErr)
 		}
 
-		limit := -1
-		if flag.Arg(1) != "" {
+		limit := 0
+		if flag.Arg(1) == "" {
+			fmt.Println("Are you sure you want to apply all down migrations? [y/N]")
+
+			reader := bufio.NewReader(os.Stdin)
+			response, err := reader.ReadString('\n')
+			if err != nil {
+				log.fatal("error: not a valid response")
+			}
+			response = strings.ToLower(strings.TrimSpace(response))
+
+			if response == "y" {
+				limit = -1
+			}
+		} else if flag.Arg(1) == "--all" {
```

Comment:
Should probably use a `FlagSet.Bool()` to parse `-all` and `--all`.
However, I'm not sure how this plays with `FlagSet.UInt()`, which is needed to specify the number of migrations to run.

Reply:
added FlagSet for `-all`, not sure if it can be used for the number of migrations the way it is currently expecting the argument. What do you think?